### PR TITLE
[Merged by Bors] - Improve issue templates

### PR DESCRIPTION
**Pull request by noamnelke**
## Motivation
Make `go-spacemesh` issue templates more useful.

## Changes
Include links to https://github.com/spacemeshos/smips and discord (to direct users there and reduce "support issue" creation), make it clear what each template is for and remove boilerplate that existed in previous templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Environment
+Please complete the following information:
+ - OS: [e.g. Windows]
+ - Node Version: [e.g. 0.1.2]
+ - App Version (if applicable): [e.g. 1.2.3]
+
+## Log files
+<!-- If possible, attach log files to this bug report by dragging and dropping them onto this text box.
+The log file is called `spacemeshlog.txt` and it is located in your documents folder. -->
```
Comment (avive) [inline .github/ISSUE_TEMPLATE/bug-report.md]: - spacemesh-log.txt
- it is not in the docs folder, it is in the spacemesh data folder. See bottom of this page: https://testnet.spacemesh.io/#ffnet
- please add recommendation to upload screen shot - for example - an error dialog box or a badly displayed data. e..g wrong balance, Nan tx nonce, etc...

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+4. See error
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Environment
+Please complete the following information:
+ - OS: [e.g. Windows]
+ - Node Version: [e.g. 0.1.2]
+ - App Version (if applicable): [e.g. 1.2.3]
```
Comment (avive) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Please add: The app version string is accessible from the app's settings.

**Review by avive — CHANGES_REQUESTED** [review]

**Review by avive — APPROVED** [review]

**Comment by noamnelke** [comment]
bors merge

**Comment by noamnelke** [comment]
bors merge

**Pull request closed without merging**